api/swagger.yaml

```diff
@@ -2102,6 +2102,16 @@ definitions:
                     SpreadDescriptor:
                       description: "label descriptor, such as engine.labels.az"
                       type: "string"
+          Platforms:
+            description: "An array of supported platforms."
+            type: "array"
+            items:
+              type: "object"
+              properties:
+                Architecture:
```

Comment:
@clnperez I'm not familiar with what a variant is, could you clarify? Also, Swarm currently only has `Architecture` and `OS` fields, which is why this structure has only those (see https://github.com/docker/swarmkit/pull/1981). We could certainly add more to make the comparison finer.